### PR TITLE
changes style of 👻 sheet costume

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
tweak: changed sprite for ghost sheet
/:cl:

the alternative style doesnt have the "hands" "up" which makes it look more realistic ghost

### CODE REVIEW

**Comment by Twaticus** [comment]
side sprite is terrifying

**Comment by Fikou** [comment]
add a toggle action button

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -724,14 +724,20 @@
...
-	inhand_icon_state = "ghost_sheet"
+	icon_state = "ghost_sheet0"
+	inhand_icon_state = "ghost_sheet0"
 	throwforce = 0
 	throw_speed = 1
 	throw_range = 2
 	w_class = WEIGHT_CLASS_TINY
 	flags_inv = HIDEGLOVES|HIDEEARS|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
 	alternate_worn_layer = UNDER_HEAD_LAYER
+	var/alt = FALSE ///var to swap icon
```
Comment (Fikou) [inline code/modules/clothing/suits/miscellaneous.dm:735]:
```suggestion
	actions_types = list(/datum/action/item_action/toggle)
	var/alt = FALSE ///var to swap icon
```

**Comment by optimumtact** [comment]
Replace the sprite, we do not need alternative sprites for a ghost sheet

**Comment by Winterous** [comment]
Don't replace it it's my sprite and I'm biased :(

Few things.
1. The front, and both side sprites are, bad. You haven't thought about the shape of it, just kind of removed the arms and left weird gaps.
2. The examine description specifically references the floating hands.

**Comment by optimumtact** [comment]
you are weak like baby

**Comment by optimumtact** [comment]
@Twaticus over to you for sprite difference

**Comment by wesoda25** [comment]
If it's a matter of replacing, the old one is definitely better. What's wrong with having two styles that you can switch between with like a wirecutter or something?

**Comment by optimumtact** [comment]
They cost bandwidth

**Comment by Twaticus** [comment]
while i dont mind it as an alternate style, oranges stated his wishes and its not enough of an upgrade to replace the original imo. no go from me

**Pull request closed without merging**